tests/unit/test_waiters.py

```diff
@@ -697,46 +697,15 @@ def assert_invalidation_completed_call_count(self, api_version=None):
         waiter.wait()
         self.assertEqual(self.client.get_invalidation.call_count, 1)
 
-    def assert_streaming_distribution_deployed_call_count(
-            self, api_version=None):
+    def assert_streaming_distribution_deployed_call_count(self):
         waiter_name = 'StreamingDistributionDeployed'
-        waiter_model = self.get_waiter_model(self.service, api_version)
+        waiter_model = self.get_waiter_model(self.service)
         self.client.meta.service_model = self.get_service_model(
-            self.service, api_version)
+            self.service)
         self.client.get_streaming_distribution.side_effect = [
             {'StreamingDistribution': {'Status': 'Deployed'}}
         ]
         waiter = create_waiter_with_client(waiter_name, waiter_model,
                                            self.client)
         waiter.wait()
         self.assertEqual(self.client.get_streaming_distribution.call_count, 1)
-
-    def test_distribution_deployed(self):
```

Comment:
For these tests, we probably only want to remove the for loop around the different API versions. The first assertion still seems valuable to make sure that those cloudfront waiters are not broken.